routes/web.php

```diff
@@ -3,12 +3,17 @@
 declare(strict_types=1);
 
 use Illuminate\Support\Facades\Route;
+use Lightit\Backoffice\Employee\App\Controllers\ListEmployeesController;
+use Lightit\Backoffice\Employee\App\Controllers\StoreEmployeeController;
 use Lightit\Shared\App\Exceptions\Http\InvalidActionException;
+use Lightit\Backoffice\Task\App\Controllers\GetTaskController;
+use Lightit\Backoffice\Task\App\Controllers\ListTasksController;
+use Lightit\Backoffice\Task\App\Controllers\UpdateTaskController;
 
 Route::get('invalid', static fn() => throw new InvalidActionException("Is not valid"));
 
-Route::get('/', static fn() => view('welcome'));
-
-Route::get('/login', static fn() => view('welcome'))->name('login');
+Route::get('/', static fn() => view('app'));
 
+Route::post('/employees', StoreEmployeeController::class)->name('employees');
 
+Route::get('/employees', ListEmployeesController::class);
```

Comment:
Add name, usually we tend to do things like `employees.post` or `employees.create`, and `employees.get` or `employees.list` so we can use `employees` in more than one